### DISCUSSION TITLE
How to add build number in history

### DISCUSSION THREAD

**Discussion by dinesh1306** [allure-framework/allure2#1864] [Questions & Support] [answered]
How to add Jenkins build id in allure report and in history?
because I have requirement where in history tab I need build Id as well

**Comment by Bhaveshgoldsmith** [comment]
You can add it in enironment details:

**Steps to do it:**

1. create environment.properties
2. add field  buildId=2.X.X
3. Keep this file in allure-results directory before report generation.

Please accept me answer if it helps you

**Comment by baev** [comment, marked as answer]
You need to provide an [executor.json](https://allurereport.org/docs/how-it-works-executor-file/) file
